### ISSUE TITLE
Make dependabot less noisy

### ISSUE THREAD

**Issue by maxammann** [maplibre/maplibre-tile-spec#286]
Does it make sense to disable dependabot?

I guess the defining question is: Is any of the code in this repo used in production?

**Comment by mactrem** [comment]
No, not yet used in production.
I think we agreed in the our last Mlt meeting, that we will run dependabot once a week instead of daily.

**Comment by springmeyer** [comment]
Yes, thanks for raising this @maxammann. Since we are targeting this being production ready I think it makes sense to keep dependabot enabled. But as @mactrem mentioned, we have recently discussed that it could be a bit less noisy on this repo. When I have time (likely mid-sept) I plan to switch it to run once a week (and also potentially bundle PRs).
